docs/logs.md

```diff
@@ -0,0 +1,67 @@
+# Cachi2 Logging Standards
+
+## Logging principles
+
+1) Use a Consistent Format: To ensure that logs are easy to read and understand, use a consistent format throughout the system. This format should include the date and time of the log entry, the severity level of the log message, and the message itself.
```

Comment:
Maybe we don't need to have the principles laid out in more general terms here. For example, I'd make this section something like:

```
1) Log format: 
[timestamp] [LEVEL] [module.path] [message] [json data]
```

I think this doc should be as specific and objective as possible. The redhat-appstudio one is a good reference for this IMO, ours should be even simpler than that.